layouts/partials/module-tabs.html

```diff
@@ -0,0 +1,54 @@
+{{ $parent := .Section }}
+<tab-panels class="c-tabs c-tabs--module">
```

Comment:
Out of curiosity why not use the shortcode here?

Reply:
Shortcodes are for markdown

Reply:
Oh of course 🤦🏻‍♂️